src/Microsoft.Health.Fhir.Shared.Api/Controllers/OperationDefinitionController.cs

```diff
@@ -33,19 +33,23 @@ public class OperationDefinitionController : Controller
         private readonly IMediator _mediator;
         private readonly OperationsConfiguration _operationConfiguration;
         private readonly FeatureConfiguration _featureConfiguration;
+        private readonly CoreFeatureConfiguration _coreFeatureConfiguration;
 
         public OperationDefinitionController(
             IMediator mediator,
             IOptions<OperationsConfiguration> operationsConfig,
-            IOptions<FeatureConfiguration> featureConfig)
+            IOptions<FeatureConfiguration> featureConfig,
+            IOptions<CoreFeatureConfiguration> coreFeatureConfig)
```

Comment:
What's the difference between `featureConfig` and `coreFeatureConfig`?

Reply:
I think the original intent was to separate fhir spec features vs fhir server features? Either way I needed to add the coreFeatureConfiguration to check the selectablesearchparameter feature flag.